pkg/informer/informer.go

```diff
@@ -454,6 +457,56 @@ func (d *DynamicDiscoverySharedInformerFactory) updateInformers() {
 		delete(d.informerStops, gvr)
 		delete(d.startedInformers, gvr)
 	}
+
+	d.discoveryData = gvrsToDiscoveryData(latest)
+}
+
+func gvrsToDiscoveryData(gvrs map[schema.GroupVersionResource]struct{}) []*metav1.APIResourceList {
```

Comment:
so this is actually fake discovery data? Can we make this clear in the go doc?

Reply:
Will do in next push

Reply:
Was merely talking about `DiscoveryData()`. Somebody will build upon it and spend lots of time to find out eventually that the APIResources are not complete. Either we make them complete, or we should call `DiscoveryData()` more like `FakeDiscoveryData()`.

Reply:
or even `QuotaDiscoveryData()`

Reply:
Will change the name, thanks